data/formations_basic.txt

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2019-2021 by Peter van der Meer
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# Formation where ships fly in a diagonal line left behind the formation leader.
+formation "Echelon (left) medium"
```

Comment:
> I think we shouldn't say 'medium' when it's the only option available. (Also, if I understand correctly, these will scale with the ships' size in the other PR, so we won't really need multiple variants.)

Automatic scaling of formations was removed from #4471, because it was implemented using quite complex keywords and a quite complex set of 4 coordinate systems. We can re-implement automatic scaling using a much simpler global scale factor if we wish, but there are currently no open PRs for automatic scaling of formations.

Having said that; removing medium from the name could also be done just because medium is the default (and then only large and small will need an additional word.)